### PR TITLE
Host release prep 4.1036.1

### PR DESCRIPTION
**Pull request by v-imohammad**
### Issue describing the changes in this PR

Updated Patch version with 4.1036.1

### Pull request checklist

**IMPORTANT**: Currently, changes must be backported to the `in-proc` branch to be included in Core Tools and non-Flex deployments.

* [ ] Backporting to the `in-proc` branch is not required
    * Otherwise: Link to backporting PR
* [x] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -19,3 +19,4 @@
 - Worker termination path updated with sanitized logging (#10367)
 - Avoid redundant DiagnosticEvents error message (#10395)
 - Added logic to shim older versions of the .NET Worker JsonFunctionProvider to ensure backwards compatibility (#10410)
+- Remove duplicate entries from the release notes (#10423)
```
Comment (amamounelsayed) [inline release_notes.md]: Please could you remove this entry? Thank you so much

**Review by amamounelsayed — APPROVED** [review]
We created this release as we created a tag 4.1036.0 early for linux dedicated. We will need this release to update the jittrace.

**Comment by kshyju** [comment]
> We created this release as we created a tag 4.1036.0 early for linux dedicated. We will need this release to update the jittrace.

Are you saying we need to generate JITTrace on top of 4.1036.0? This PR is bumping the patch version. I do not see any changes to update the JITTrace in this. What is the reason for bumping the patch version?

**Comment by fabiocav** [comment]
Have a similar question to @kshyju's above. Can you share more context on the patch bump?

**Comment by amamounelsayed** [comment]
> > We created this release as we created a tag 4.1036.0 early for linux dedicated. We will need this release to update the jittrace.
> 
> Are you saying we need to generate JITTrace on top of 4.1036.0? This PR is bumping the patch version. I do not see any changes to update the JITTrace in this. What is the reason for bumping the patch version?

Thank you so much, here is the PR https://github.com/Azure/azure-functions-host/commit/7017726726a38dadc4a1232a7522db4941b06377

**Review by kshyju — APPROVED** [review]
Discussed offline. We decided to proceed with this approach for this instance.

**Pull request merged**